src/js/media-controller.js

```diff
@@ -45,6 +48,13 @@ const {
   MEDIA_PLAYBACK_RATE_REQUEST,
 } = MediaUIEvents;
 
+const ButtonPressedKeys = ['Enter', ' ', 'f', 'm', 'k', 'ArrowLeft', 'ArrowRight'];
+const DEFAULT_SEEK_OFFSET = 10;
+
+/**
+ * Media Controller should not mimic the HTMLMediaElement API.
+ * @see https://github.com/muxinc/media-chrome/pull/182#issuecomment-1067370339
+ */
```

Comment:
moved from the bottom of the class here, which seems more appropriate.

Reply:
![](https://media.giphy.com/media/TGamxxMMiaWw8/giphy.gif)